### PR TITLE
[BUG] Fix hadoop ai build path

### PR DESCRIPTION
**Pull request by mzmssg**
Issue link: #2021

### CODE REVIEW

File: src/hadoop-ai/build/build-pre.sh [hunk 1]
```diff
@@ -42,7 +42,7 @@ echo "hadoopbinarypath:${hadoopBinaryDir}"
 }
 
 # When Changing the patch id, please update the filename here.
-rm ${hadoopBinaryDir}/*-done
+rm ${hadoopBinaryDir}*-done
```
Comment (ydye) [inline src/hadoop-ai/build/build-pre.sh]: For which case, the removed line won't work?

Comment (mzmssg) [reply src/hadoop-ai/build/build-pre.sh]: @ydye 

> 2019-03-04 08:08:40,342 - core.build_utility - INFO - Executing command successfully: chmod u+x src/hadoop-ai/build/build-pre.sh
2019-03-04 08:08:40,342 - core.build_utility - INFO - Begin to execute the command: src/hadoop-ai/build/build-pre.sh
hadoopbinarypath:/hadoop-binary/
**rm: cannot remove '/hadoop-binary//*-done': No such file or directory**
Sending build context to Docker daemon   16.9kB
Step 1/8 : FROM ubuntu:16.04
 ---> a51debf7e1eb
Step 2/8 : RUN apt-get -y update &&     apt-get -y install       nano       vim       joe       wget       curl       jq       gawk       psmisc       python       python-yaml       python-jinja2       python-urllib3       python-tz       python-nose       python-prettytable       python-netifaces       python-dev       python-pip       python-mysqldb       openjdk-8-jre       openjdk-8-jdk       openssh-server       openssh-client       git       inotify-tools       rsync       maven       cmake       findbugs       zlib1g-dev       pkg-config       libssl-dev       autoconf       automake       libtool       build-essential
 ---> Using cache
 ---> 6dd6935795ab

Comment (mzmssg) [reply src/hadoop-ai/build/build-pre.sh]: Seems misleaded by the error message:
>hadoopbinarypath:/hadoop-binary/
*rm: cannot remove '/hadoop-binary//*-done': No such file or directory*
Sending build context to Docker daemon   16.9kB

Confirmed that double slashes should work. Need to investigate more about origin issue.

Whatever, removing redundent slash isn't bad.

Comment (ydye) [reply src/hadoop-ai/build/build-pre.sh]: Can this change handle this path in shell script? 
```
/hadoop-binary/*-done
```

**Pull request merged**